Makefile

```diff
@@ -0,0 +1,26 @@
+# It is reccomended to start 3 terminal tabs/windows. One for each of the following:
+#   - build-frontend
+#   - start-database
+#   - start-server
+# That way, you can monitor potential errors in all areas.
+
```

Comment:
update it somehow. if you only execute `make` it will run `build-frontend`. either make an empty command or start-server (in my opionion)

Reply:
Only executing make is not the intended usage, I think that's not that big of a deal